test/ask/respondents_filter_test.exs

```diff
@@ -1,4 +1,4 @@
-defmodule Ask.RespondentsFilterTest do
```

Comment:
Ups, sorry, @ysbaddaden, I think I changed this (in this file and others) by error while merging changes from `main` into this branch 🤦 . Do you want to review this, or do you prefer I fix it before merging it?